doc/cephfs/troubleshooting.rst

```diff
@@ -382,36 +382,53 @@ Please see: https://github.com/ceph/ceph/blob/master/src/script/kcon_all.sh
 In-memory Log Dump
 ==================
 
-In-memory logs can be dumped by setting ``mds_extraordinary_events_dump_interval``
-during a lower level debugging (log level < 10). ``mds_extraordinary_events_dump_interval``
-is the interval in seconds for dumping the recent in-memory logs when there is an Extra-Ordinary event.
+Logs that are in memory can be dumped by setting
+``mds_extraordinary_events_dump_interval`` during a lower level debugging (when
```

Comment:
‘debugging is a noun’?  

I would take out

during a lower level debugging (

and the )